lib/src/progress-bar/ProgressBar.tsx

```diff
@@ -20,13 +27,12 @@ const DxcProgressBar = ({ label = "", helperText = "", overlay = true, value, sh
               {label}
             </ProgressBarLabel>
             <ProgressBarProgress overlay={overlay} showValue={showValue} backgroundType={backgroundType}>
-              {value === "" ? 0 : value >= 0 && value <= 100 ? value : value < 0 ? 0 : 100} %
+              {value === null ? 0 : value >= 0 && value <= 100 ? value : value < 0 ? 0 : 100} %
             </ProgressBarProgress>
           </InfoProgressBar>
           <LinearProgress
             variant={showValue ? "determinate" : "indeterminate"}
-            value={value === "" ? 0 : value >= 0 && value <= 100 ? value : value < 0 ? 0 : 100}
-            helperText={helperText}
```

Comment:
Why do you removed this prop?

Reply:
Because helperText prop doesn't exist in LinearProgress.